badparking/mobile_api/views.py

```diff
@@ -1,3 +1,40 @@
-from django.shortcuts import render
+from django.http import Http404
+from django.contrib.auth.models import User
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
 
-# Create your views here.
+from core.models import Claim, CrimeType
+from media.models import MediaFileModel
+from .serializers import ClaimSerializer
+
+
+class ClaimList(APIView):
+    def get(self, request, format=None):
```

Comment:
This should filtered by user that own the claims. We don't want to show all claims to everyone.